pkg/cmd/pr/create/create.go

```diff
@@ -287,11 +289,15 @@ func createRun(opts *CreateOptions) error {
 			return fmt.Errorf("could not compute title or body defaults: %w", defaultsErr)
 		}
 	} else if opts.Autofill {
-		if defaultsErr != nil {
+		if defaultsErr != nil && !(titleProvided || bodyProvided) {
```

Comment:
I'm not 100% sure this is 'correct'. When we fail to compute defaults, can we tell which of title or body failed to compute? If that's possible, then we could make this more precise and fail for cases such as

- couldn't compute default title && !titleProvided
- couldn't compute default body && !bodyProvided